Not working with MVC 5 and packages.config

### Functional impact
The RestoreTask cannot be loaded due to wrong relative paths in the targets file and hence the compilation of the project fails. 

### Minimal repro steps

1. Create a (old style csproj) ASP.net MVC application
2. Install the Microsoft.Web.LibraryManager.Build nuget package  
3. Create a libman.json in the project root 
4. Compile the project. 
5. Notice the compilation error. 


### Expected result
The restore task is loaded correctly and the restore is done in case the LibraryRestore property is set. 

### Actual result
A build error appears: 
```
D:\Dev\Git\MyProject\packages\Microsoft.Web.LibraryManager.Build.2.0.48\build\Microsoft.Web.LibraryManager.Build.targets(35,9): error MSB4062: The "Microsoft.Web.LibraryManager.Build.RestoreTask" task could not be loaded from the assembly C:\Users\myuser\.nuget\packages\microsoft.web.librarymanager.build\1.0.172\build\..\tools\net472\Microsoft.Web.LibraryManager.Build.dll. Could not load file or assembly 'file:///C:\Users\myuser\.nuget\packages\microsoft.web.librarymanager.build\1.0.172\tools\net472\Microsoft.Web.LibraryManager.Build.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
```

### Further technical details
The following 2 lines must not use relative paths but use the`$(MSBuildThisFileDirectory)` property to navigate relative to the own file. Just like the [Microsoft.Web.LibraryManager.Build.props](https://github.com/aspnet/LibraryManager/blob/776139c943e8c980f6656a3ae1fc5cf14efa6bc6/src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.props#L5) does it.  

https://github.com/aspnet/LibraryManager/blob/776139c943e8c980f6656a3ae1fc5cf14efa6bc6/src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.targets#L5-L6

--- a/src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.targets
+++ b/src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.targets
@@ -2,8 +2,8 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
     <PropertyGroup>
-        <_LibraryTaskAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">..\tools\netstandard2.0\Microsoft.Web.LibraryManager.Build.dll</_LibraryTaskAssembly>
-        <_LibraryTaskAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">..\tools\net472\Microsoft.Web.LibraryManager.Build.dll</_LibraryTaskAssembly>
+        <_LibraryTaskAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\netstandard2.0\Microsoft.Web.LibraryManager.Build.dll</_LibraryTaskAssembly>
+        <_LibraryTaskAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\Microsoft.Web.LibraryManager.Build.dll</_LibraryTaskAssembly>
     </PropertyGroup>
 
     <PropertyGroup>

